### PR TITLE
Synthanol Upgrade v2

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Synthanol will now notify organic users of how awfully it tastes and make them puke. No toxin damage or anything inherently damaging. It will get purged very fast.

Also adds a few new IPC drinks and makes Holy Water have its own glass, making it actually indistinguishable from a water glass (used to turn into a "Glass of.. what?")

MOAR BOOZE
:cl:
add: Adds three new synthanol drinks
tweak: Drinking synthanol is now a bad idea if you're organic
tweak: A glass of holy water now looks like normal water
/:cl:

This is the second version of this PR, no merge conflicts this time. Hopefully.
*fizz*

### CODE REVIEW

**Comment by Ar3nn** [comment]
rip

File: code/modules/reagents/newchem/drinks.dm [hunk 1]
```diff
@@ -49,7 +49,7 @@
 	max_temp = 270
 	result_amount = 1
 	mix_message = "The drink darkens as the water freezes, leaving the concentrated cider behind."
-	mix_sound = null
+	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
```
Comment (Fox-McCloud) [inline code/modules/reagents/newchem/drinks.dm]: This one is meant to be null as it's just freezing the cider.

File: code/modules/reagents/newchem/drinks.dm [hunk 2]
```diff
@@ -96,16 +97,34 @@
...
+		holder.remove_reagent(id, 3.6) //gets removed from organics very fast
+		if(prob(25))
+			holder.remove_reagent(id, 15)
+			M.fakevomit()
 	..()
 
+datum/reagent/ethanol/synthanol/reaction_mob(var/mob/M, var/method=TOUCH, var/volume)
+	if(!istype(M, /mob/living))
+		return
+	if(M.get_species() == "Machine")
```
Comment (Crazylemon64) [inline code/modules/reagents/newchem/drinks.dm]: Might want an `issynthetic()` instead.

File: code/modules/reagents/newchem/drinks.dm [hunk 3]
```diff
@@ -154,4 +173,48 @@
...
+	reagent_state = LIQUID
+	color = "#7204e7"
+
+/datum/chemical_reaction/synthanol/synthnsoda
+	name = "Synth 'n Soda"
+	id = "synthnsoda"
+	result = "synthnsoda"
+	required_reagents = list("synthanol" = 1, "cola" = 1)
+	result_amount = 2
+	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
```
Comment (Crazylemon64) [inline code/modules/reagents/newchem/drinks.dm]: Perhaps you could move this up into the `/datum/chemical_reaction/synthanol` class, so you don't have to constantly re-define it?

File: code/modules/reagents/newchem/drinks.dm [hunk 4]
```diff
@@ -96,16 +97,34 @@
...
+			var/datum/effect/system/spark_spread/s = new /datum/effect/system/spark_spread
+			s.set_up(3, 1, M)
+			s.start()
+		if(d >= braindamage_start && prob(33))
+			M.adjustBrainLoss(1)
+
+	else
+		holder.remove_reagent(id, 3.6) //gets removed from organics very fast
+		if(prob(25))
+			holder.remove_reagent(id, 15)
```
Comment (VampyrBytes) [inline code/modules/reagents/newchem/drinks.dm]: Could these removes lead to a negative amount of synthanol in the mob? maybe find out how much is in them and then use a min to make sure they don't go below 0

Comment (FlattestGuitar) [inline code/modules/reagents/newchem/drinks.dm]: `remove_reagent` is what's already used for every single reagent that gets metabolized. I tested this and there's no way the reagent can hit negative values using this.

Comment (VampyrBytes) [inline code/modules/reagents/newchem/drinks.dm]: My bad, I'd glanced over `remove_reagent` and not seen a safety net in there, I'd missed the call to 'update_total` though, and that's got the safety net
Sorry :blush:

File: code/modules/reagents/reagent_containers/food/drinks/drinkingglass.dm [hunk 5]
```diff
@@ -580,6 +580,22 @@
...
+					name = "Glass of Synthignon"
+					desc = "Someone mixed good wine and robot booze. Romantic, but atrocious."
+				if("uplink")
+					icon_state = "uplinkglass"
+					name = "Glass of Uplink"
+					desc = "An exquisite mix of the finest liquoirs and synthanol. Meant only for synthetics."
+				if("holywater")
+					icon_state = "glass_clear"
+					name = "Glass of Holy Water"
+					desc = "The contents of this glass seem to be filled with an energy of some kind."
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/food/drinks/drinkingglass.dm]: The name and description should be identical to water--just like the reagent's name+description.

**Pull request merged**